docs/Using Randomness/vrf-best-practices.md

```diff
@@ -15,71 +15,77 @@ metadata:
 >
 > If you are using v1, see the [VRF v1 guide](./v1).
 
-Best are the practices for using Chainlink VRF.
+These are the best practices for using Chainlink VRF.
 
 ## Getting a random number within a range
 
-If you need to generate a random number within a given range, you should use [modulo](https://docs.soliditylang.org/en/v0.8.7/types.html#modulo) to define the limits of your range. Below you can see how to get a random number in a range from 1 to 50.
+If you need to generate a random number within a given range, use [modulo](https://docs.soliditylang.org/en/v0.8.7/types.html#modulo) to define the limits of your range. Below you can see how to get a random number in a range from 1 to 50.
 
 ```solidity
-uint256 public randomResult;
-
-function fulfillRandomness(uint256 requestId, uint256[] randomness) internal override {
-    // Assuming only one random word was requested.
-    randomResult = (randomness[0] % 50) + 1;
+function fulfillRandomWords(
+  uint256, /* requestId */
+  uint256[] memory randomWords
+) internal override {
+  // Assuming only one random word was requested.
+  s_randomRange = (randomWords[0] % 50) + 1;
 }
 ```
 
 ## Getting multiple random values
 
 If you want to get multiple random values from a single VRF request, you can request this directly with the `numWords` argument. See the [Get a Random Number](/docs/get-a-random-number/) guide for an example where one request returns multiple random values.
 
-## Having multiple VRF requests in flight
+## Processing simultaneous VRF requests
 
-If you want to have multiple VRF requests in flight, you might want to create a mapping between the `requestId` and the address of the requester.
+If you want to have multiple VRF requests processing simultaneously, create a mapping between `requestID` and the response. You might also create a mapping between the `requestId` and the address of the requester to track the most recent request that each address made. This way you can find the request ID for the requesting address and use that ID to look up the most recent stored response.
 
 ```solidity
-mapping(uint256 => address) public requestIdToAddress;
-
-function getRandomNumber() public {
-    uint256 requestId =  COORDINATOR.requestRandomWords(keyHash, subId, 10, 200000, 1);
-    requestIdToAddress[requestId] = msg.sender;
+mapping(address => uint256) public s_AddressTorequestId;
+mapping(uint256 => uint256[]) public s_randomWordsTorequestId;
+
+function requestRandomWords() external onlyOwner {
+  uint256 requestId = COORDINATOR.requestRandomWords(
+    keyHash,
+    s_subscriptionId,
+    requestConfirmations,
+    callbackGasLimit,
+    numWords
+  );
+  s_AddressTorequestId[msg.sender] = requestId;
```

Comment:
From the above paragraph I thought you'd want to map requestId to the caller address. This map wouldn't help you figure out which requester a certain fulfillment was for I think

Reply:
I reversed the mapping based on Makram's feedback. That makes more sense now.